docs/src/guide/basics.rst

```diff
@@ -136,7 +136,7 @@ Handles are opaque structs named as ``uv_TYPE_t`` where type signifies what the
 handle is used for. 
 
 .. rubric:: libuv watchers
-.. literalinclude:: ../libuv/include/uv.h
+.. literalinclude:: ../../../../libuv/include/uv.h
```

Comment:
Looks like just an artifact of how https://github.com/nikhilm/uvbook is structured.